src/test/java/org/wso2/lsp4intellij/utils/FileUtilsTest.java

```diff
@@ -0,0 +1,180 @@
+package org.wso2.lsp4intellij.utils;
+
+import com.intellij.openapi.editor.Document;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.fileEditor.FileDocumentManager;
+import com.intellij.openapi.fileEditor.FileEditor;
+import com.intellij.openapi.fileEditor.FileEditorManager;
+import com.intellij.openapi.fileEditor.TextEditor;
+import com.intellij.openapi.fileEditor.ex.FileEditorManagerEx;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.testFramework.BinaryLightVirtualFile;
+import com.intellij.testFramework.LightVirtualFile;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.io.File;
+import java.net.URISyntaxException;
+
+@RunWith(PowerMockRunner.class)
+public class FileUtilsTest {
+
+  @PrepareForTest(FileEditorManager.class)
+  @Test
+  public void testEditorFromVirtualFile() {
+    VirtualFile file = PowerMockito.mock(VirtualFile.class);
+    Project project = PowerMockito.mock(Project.class);
+    
+    Editor editor = PowerMockito.mock(Editor.class);
+    TextEditor textEditor = PowerMockito.mock(TextEditor.class);
+    PowerMockito.when(textEditor.getEditor()).thenReturn(editor);
+
+    FileEditorManagerEx fileEditorManagerEx = PowerMockito.mock(FileEditorManagerEx.class);
+
```

Comment:
Remove blank line.. try to keep the mocking for the object defined in one block.. its easier to read :)